### DISCUSSION TITLE
Address bar always takes 2 rows or more

### DISCUSSION THREAD

**Discussion by Anixx** [valinet/ExplorerPatcher#250] [Q&A]
The Address bar is huge and always occupies the whole vertical space of the taskbar. It also makes the taskbar 2-row even if it was 1-row before adding the address bar. I cannot make the address bar smaller:
![изображение](https://user-images.githubusercontent.com/1271736/139636686-7bace0eb-7ea9-4b01-9909-d946295fdc8f.png)
I have seen on screenshots of others that they somehow make it small or even put it into the top or bottom row.

**Comment by valinet** [comment]
It’s the same problem as with Explorer’s address bar, probably… what can I do? I already replied about that on the other thread without seeing this first. They made that thicker since 1903 so it’s easier to hit by touch users… if one figures out how that height is consumed, maybe changing it will make the thing behave normally again… idk, again, has to be studied yet again more than a little bit.

**Comment by Anixx** [comment]
I do not have a touch screen! So it should be small! Their check for the equipment is broken!

**Reply by valinet** [reply]
I think there’s no check. They made it bigger regardless.

To be honest, I have large screens, high DPI; for me, with the default theme, I actually prefer it like this since it’s easier to use. And believe me, when you work with a large screen, you get tired of apps not scaling properly and presenting you with tiny glyphs.

Plus, I think you’re exaggerating a fair bit. Why use the address bar in the taskbar anyway…? And even in Explorer, use the regular one and maybe some time, either someone discovers how to workaround and get the tiny one back, or maybe someone develops a BHO that recreates the address bar and calls it a day…
